pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```

Comment:
I don't think we need/should require wheels, falling back to sdist is perfectly fine.
(Yeah I know it's the default snippet, but I don't like adding dependencies we don't need)

Reply:
Again, I don't think it's worth deviating from the standard here.  As you note setuptools tells you do this [1].  I suspect (although I'm not certain), it's so "pip wheel" just works and doesn't fail with odd errors.  For example our docker build process has a "builder" phase that produces wheels in the base environment, that are then installed into our production images so they don't have build dependencies.  This is just to say it's not an odd or specific thing to be making wheels these days.  I think the project will benefit more from matching the status quo.

[1] https://setuptools.readthedocs.io/en/latest/build_meta.html

Reply:
This is a very recent new addition, their documentation didn't even mention it a year ago: https://web.archive.org/web/20200720013741/https://setuptools.readthedocs.io/en/latest/setuptools.html

setuptools will be fine without it.

Reply:
It seems to me that PEP 517 requires a build_wheel entry point, so a compliant [build-system] section needs to be able to implement that.  I imagine that is why it is in there.  It would probably be good to clarify this

Reply:
I see. Can we get rid of `[build-system]` entirely and just use `setup.py` like we used to before PEP 517, then?

Reply:
The project already has a ```pyproject.toml``` but without ```build-system``` and so, per [1], pip is already assuming it uses setuptools/wheel

> Projects with a pyproject.toml file will use a PEP 517 backend. Projects with a pyproject.toml file, but which don’t have a build-system section, will be assumed to have the following backend settings:
> [build-system]
> requires = ["setuptools>=40.8.0", "wheel"]
> build-backend = "setuptools.build_meta:__legacy__"

All this is doing is being explicit about it.

[1] https://pip.pypa.io/en/stable/cli/pip/#pep-517-and-518-support

Reply:
ouch, ok then